pages/courses/resources/index.tsx

```diff
@@ -92,13 +92,18 @@ export default function Resources() {
                                 </button>
                             </div>
 
-                            <div className="flex flex-col md:flex-row w-1/3 justify-center">
+                            <div className="flex flex-col hidden md:block md:flex-row w-1/3 justify-center">
                                 <Link className="m-3 w-full" href={"/courses/resources/add"}>
                                     <button className="btn btn-outline w-full">
                                         Add a Resource
                                     </button>
                                 </Link>
                             </div>
+                            <div className="z-10 w-14 fixed bottom-5 right-0 m-4 cursor-pointer md:hidden">
+                                <Link className="m-3 w-full" href={"/courses/resources/add"}>
```

Comment:
what is the color of the plus? once check rants page, just in case

Reply:
Earlier it was grey, but yes, it looks bad
Changed to cyan, like the rants page now